src/npm-fastui-bootstrap/src/footer.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react'
+import { components, useClassName } from 'fastui'
+
+export const Footer: FC<components.FooterProps> = (props) => {
+  const links = props.links.map((link) => {
+    link.mode = link.mode || 'footer'
+    return link
+  })
+  const extraSeparatorProp = useClassName(props, { el: 'extra-separator' })
```

Comment:
we should remove the horizontal line between the links and the text.

Reply:
Done